### PR TITLE
Lint Ansible roles also in CI

### PR DESCRIPTION
**Pull request by sauloperez**
This makes ansible-lint check roles besides playbooks.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ python: 3.7.4
 jobs:
   include:
     - stage: lint
-      script: ansible-lint playbooks/*.yml
+      script: ansible-lint playbooks/**/*.yml roles/**/*.yml
```
Comment (enricostano) [inline .travis.yml]: From my experience this is already done by `playbooks/*.yml`. Don't ask me why :grimacing:

Comment (enricostano) [reply .travis.yml]: Look at this output from `odoo-provisioning` repo: https://gist.github.com/enricostano/d4822a28e7db2b70d5f0735005388c0a

Comment (sauloperez) [reply .travis.yml]: I have the hypothesis that including the role makes it read the role's code as the `ansible` command does but not all. Check yourself:

```
➜  timeoverflow-provisioning install-es-2.4.6 ✗ ansible-lint roles/**/*.yml                                                                                                                                                                        <<<
[206] Variables should have spaces before and after: {{ var_name }}
roles/elasticsearch/tasks/main.yml:13
  template: src=elasticsearch.j2 dest={{instance_sysd_script}} mode=0644 force=yes

[206] Variables should have spaces before and after: {{ var_name }}
roles/elasticsearch/vars/main.yml:3
instance_sysd_script: "{{sysd_script | dirname }}/{{es_instance_name}}_{{sysd_script | basename}}"
```

```
➜  timeoverflow-provisioning install-es-2.4.6 ✗ ansible-lint playbooks/*.yml
[206] Variables should have spaces before and after: {{ var_name }}
/home/pau/dev/timeoverflow-provisioning/roles/elasticsearch/tasks/main.yml:13
  template: src=elasticsearch.j2 dest={{instance_sysd_script}} mode=0644 force=yes
```

Comment (enricostano) [reply .travis.yml]: > but not all

I see, but I don't understand :see_no_evil: It's a bug? I any case, go ahead since "more is better"...

**Comment by sauloperez** [comment]
Fail :trollface: 

```
$ ansible-lint playbooks/**/*.yml roles/**/*.yml
WARNING: Couldn't open roles/**/*.yml - No such file or directory
WARNING: Couldn't open playbooks/**/*.yml - No such file or directory
The command "ansible-lint playbooks/**/*.yml roles/**/*.yml" exited with 0.
```

could it be a bash thing as @enricostano mentioned offline? I run zsh on my laptop.

File: .travis.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ python: 3.7.4
 jobs:
   include:
     - stage: lint
-      script: ansible-lint playbooks/*.yml
+      script: shopt -s globstar && ansible-lint playbooks/**/*.yml roles/**/*.yml
```
Comment (enricostano) [inline .travis.yml:9]: @raneq FYI :point_up:

**Comment by enricostano** [comment]
```
$ shopt -s globstar && ansible-lint playbooks/**/*.yml roles/**/*.yml
206 Variables should have spaces before and after: {{ var_name }}
roles/elasticsearch/vars/main.yml:3
instance_sysd_script: "{{sysd_script | dirname }}/{{es_instance_name}}_{{sysd_script | basename}}"
```
Almost there! :muscle:

**Comment by sauloperez** [comment]
I'd like to understand why we don't see ansible-lint's output colored 🤔

File: .travis.yml (hunk 2 above)
Comment (raneq) [reply .travis.yml:9]: man bash → /globstar

```
 *      Matches any string, including the null string.  When the globstar shell option is enabled, and * is used in a pathname expansion  context,  two
       adjacent *s used as a single pattern will match all files and zero or more directories and subdirectories.  If followed by a /, two adjacent *s
       will match only directories and subdirectories.

```

**Pull request closed without merging**